tests/unit/amo/components/TestCollection.js

```diff
@@ -145,6 +161,21 @@ describe(__filename, () => {
     expect(wrapper.find('.Collection-placeholder')).toHaveLength(0);
   });
 
+  it('hides placeholder text if the user is not logged in', () => {
```

Comment:
Isn't it a duplicate test? If user is not logged in, there is no placeholder text. The other test `it hides placeholder text when creating a collection if not logged in'` already covers this case, no?

Reply:
There's only one code path: you are correct that if a user is not logged in there will be no placeholder, regardless of any other circumstances, but it's a different use case. The earlier test is for the "create" use case, while this is for the "display" use case. We know the code, so we can say it's testing the same thing, but is it still worth having tests for the 2 different use cases? If someone were to change the implementation and we didn't have this case it's possible that a bug could be introduced. I'll update the description to make it more obvious that it's different than the "create" test.